components/molecules/ContributorProfileInfo/contributor-profile-info.tsx

```diff
@@ -64,14 +68,14 @@ const ContributorProfileInfo = ({
                 "I am an open source developer with a passion for music and video games. I strive to improve the open source community and am always looking for new ways to contribute."}
             </p>
             <div className="flex flex-col text-sm mt-2 text-light-slate-9 gap-2">
-              <span className="flex gap-2 items-center">
+              {displayLocalTime && <span className="flex gap-2 items-center">
                 <FiClock className="text-light-slate-9" /> Local time: 12:32 PM
-              </span>
+              </span>}
 
               <span className="flex gap-2 items-center">
                 <FiTwitter className="text-light-slate-9" />
                 <Link href="#" className="w-max hover:text-orange-500 ">
-                  coming soon...
+                  {twitterUsername || "coming soon..."}
```

Comment:
Do we still need the coming soon here? IMHO, this should be off if a user twitter username is null and display when a it's available for the user 🍕

Reply:
It should not show if it doesn't exist. Coming soon is not needed.


Also, TBH. I never meant for this to show on the user profile. It was meant for the settings side as a placeholder.

Reply:
Fixed